scss/titles.scss

```diff
@@ -0,0 +1,20 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title {
+  color: $black;
+  font-size: $font-title-size;
+  margin-top: 50px;
+
+  &::before{
+    content: url(assets/triangle.svg);
+    display: block;
+    float: left;
+    height: $title-decoration-height;
+    left: 19px;
+    position: relative;
+    top: -35px; //TODO: how to do it without negative values?
```

Comment:
you can do this by adding a padding left to the title, then the absolute will start from the padding 😉